demisto_sdk/tests/test_files/content_slim/Packs/Sample02/Layouts/layout-sample2_packs.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
`layout` is only for 5.5 and below.
should be `layoutscontainer`

Reply:
Changed